CONTRIBUTING.md

```diff
@@ -0,0 +1,92 @@
+# Contributing
+
+When contributing to this repository, please first discuss the change you wish to make via issue,
+email, or any other method with the owners of this repository before making a change. 
+
+Please note we have a code of conduct, please follow it in all your interactions with the project.
+
+## Pull Request Process
+
+1. Ensure any install or build dependencies are removed before the end of the layer when doing a 
```

Comment:
"before the end of the layer"? I don't know what this means.

Reply:
It means the last commit. Sometimes people commit stuff in the middle that needs to be removed or cleaned.